src/web/make-graph.rkt

```diff
@@ -201,18 +213,18 @@
              (ol ([class "history"])
                  ,@(render-history alt train-pctx test-pctx ctx)))))
 
-      ,(if (test-output test)
-           (let-values ([(dropdown body) (render-program (test-output test) ctx #:ident identifier)])
-             `(section ([id "target"] [class "programs"])
-                       (h2 "Developer target"
-                           ": "
-                           (span ([class "subhead"])
-                                 (data ,(format-accuracy (errors-score target-error) repr-bits #:unit "%")) " accurate, "
-                                 (data ,(~r (/ (alt-cost start-alt repr) target-cost) #:precision '(= 1)) "×") " speedup")
-                           ,dropdown
-                           ,(render-help "report.html#target"))
-                       ,body))
-           "")
 
+      ,@(for/list ([i (in-naturals 1)] [target target-alt-list] [target-error list-target-error] [target-cost list-target-cost])
+          (let-values ([(dropdown body) (render-program (alt-expr (alt-analysis-alt target)) ctx #:ident identifier)])
+            `(section ([id "target"] [class "programs"])
```

Comment:
It doesn't make sense to give multiple elements the same `id`. Do we use this id anywhere? If so, we need to fix it. Possibly we should give them IDs like `target1`, `target2`, etc.